r2ai/repl.py

```diff
@@ -78,6 +79,7 @@ def do_POST(self):
 help_message = """Usage: r2ai [-option] ([query] | [script.py])
  r2ai . [file]          interpret r2ai script with access to globals
  r2ai :aa               run a r2 command
+ r2ai :auto [prompt]    query LLM that can interact with r2 (WIP, OpenAI only atm)
```

Comment:
thats the only thing i dislike from this PR . the : character is suposed to run r2 commands. and auto is not an r2 command. it will be better to use a different thing like adding a new flag (-a and -A are taken already, but we can change them if needed), or we can use the "' " syntax as you did in r2d2, the "'" thing conflicts with the r2 syntax, but as long as there's a space after the quote it is fine for me to use it that way.

i will merge it the way it is because i dont think this is a blocker for anything, and we have plenty of time to discuss that later. :)

Reply:
makes sense, i didn't even think about it when i started with ":auto", i'll change it to "' " in the next round